charts/templates/statefulset.yaml

```diff
@@ -23,8 +24,104 @@ spec:
       affinity: {{- toYaml . | nindent 8}}
       {{- end }}
       terminationGracePeriodSeconds: {{ .Values.terminationGracePeriod }}
-      {{- if ne .Values.bootstrapHostName "" }}
+      {{- if or ( ne .Values.bootstrapHostName "") (eq .Values.tls.enableOnDefaultAppServers true ) }}
       initContainers: 
+      {{- if eq .Values.tls.enableOnDefaultAppServers true }}
+      - name: copy-certs
+        image: "finalgene/openssh"
+        command:
+          - bash
+          - '-c'
+          - |
+            MARKLOGIC_ADMIN_USERNAME="$(< /run/secrets/ml-secrets/username)"            
+            MARKLOGIC_ADMIN_PASSWORD="$(< /run/secrets/ml-secrets/username)"
+            if [[ -d "/tmp/server-cert-secrets" ]]; then
+              certType="named"
+            else
+              certType="self-signed"
+            fi
+            echo "Proceeding with $certType certificate flow."
```

Comment:
We should use the same log function that we are using in other Initcontainer and post-start hook

            log () {
                      local TIMESTAMP=$(date +"%Y-%m-%d %T.%3N")
                      echo "${TIMESTAMP}  $@"
                    }

Reply:
It's fixed.